roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    description: Prometheus
+
+# secrets
+- name: Set alert and prometheus secrets
+  oc_secret:
+    state: "{{ state }}"
+    name: "{{ item }}-proxy"
```

Comment:
All of these names need to be consistent with the openshift origin template.  If we want to further prefix them, they should be prefixed in the origin template (i.e. I don't want drift in how the version in origin works to how the one installed by ansible works).